codegens/r-rcurl/test/unit/parseBody.test.js

```diff
@@ -0,0 +1,204 @@
+var expect = require('chai').expect,
+  sdk = require('postman-collection'),
+  fs = require('fs'),
+  path = require('path'),
+  {
+    parseURLEncodedBody,
+    parseBody,
+    parseFormData,
+    parseRawBody,
+    parseGraphQL,
+    buildFormDataParamFile
+  } = require('../../lib/util/parseBody'),
+  collectionsPath = './fixtures';
+
+describe('parseURLEncodedBody method', function () {
+  it('should return form-url-encoded params', function () {
+    const collection = new sdk.Collection(JSON.parse(
+        fs.readFileSync(path.resolve(__dirname, collectionsPath, './sample_collection.json').toString()))),
+      body = collection.items.members[7].request.body.urlencoded,
+      indentation = '  ',
+      bodyTrim = false,
+      expectedBody = 'c(\n' +
```

Comment:
@LuisTejedaS Why are we adding so many escape characters?

Reply:
Can you post how this would look like when viewed in an editor?

Reply:
Sure @umeshp7  this is the generated snippet with this example, double quotes are preferred in R so here we are escaping them, for this expected value as we are using JS and our preferred are single quotes we have to escape single quotes and we have to scape slashes too, but this is something that we have to do only in our expected JS strings.
Hope this answers the question.

![Screen Shot 2022-04-04 at 10 04 16](https://user-images.githubusercontent.com/46000487/161574122-0a61c06b-2305-4af5-9341-bffeeb65894d.png)